src/ClientServices/ProcessClient.cpp

```diff
@@ -6,6 +6,7 @@
 
 #include <absl/types/span.h>
 #include <grpcpp/grpcpp.h>
+#include <grpcpp/impl/codegen/status_code_enum.h>
```

Comment:
Do you need to include this "impl" file?

Reply:
no. I removed it